script/nsis/installer.nsi

```diff
@@ -0,0 +1,53 @@
+Name    "Git LFS"
+OutFile "GitLFSInstaller.exe"
+
+
+InstallDir $DESKTOP
+
+VIProductVersion "0.6.0.0"
+VIAddVersionKey ProductName      "Git LFS"
+VIAddVersionKey Comments         "Git LFS tool"
+VIAddVersionKey CompanyName      "GitHub, Inc"
+VIAddVersionKey LegalCopyright   "GitHub, Inc"
+VIAddVersionKey FileDescription  "Git LFS Tool"
+VIAddVersionKey FileVersion      1
+VIAddVersionKey ProductVersion   1
+VIAddVersionKey InternalName     "Git LFS"
+VIAddVersionKey LegalTrademarks  "something, something, darkside"
+VIAddVersionKey OriginalFilename "GitLFSInstaller.exe"
```

Comment:
These are probably ALL WRONG :) Need to figure out how to sign this properly, so Windows can stop saying it's coming from an "unknown publisher".